### ISSUE TITLE
Docker won't start since last pull ( 3.3.9 )

### ISSUE THREAD

**Issue by MervinJMinky** [FlareSolverr/FlareSolverr#964]
### Have you checked our README?

- [X] I have checked the README

### Have you followed our Troubleshooting?

- [X] I have followed your Troubleshooting

### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Have you checked the discussions?

- [X] I have read the Discussions

### Environment

```markdown
- FlareSolverr version: 3.3.9
- Last working FlareSolverr version: 3.3.7
- Operating system:unraid
- Are you using Docker: [yes/no]yes
- FlareSolverr User-Agent (see log traces or / endpoint):
- Are you using a VPN: [yes/no]no
- Are you using a Proxy: [yes/no]no
- Are you using Captcha Solver: [yes/no]no
- If using captcha solver, which one:
- URL to test this issue:private
```

### Description

Hello, 
I cannot get my flaresolverr docker to start since last update.
It works fine if i roll back to 3.3.7 but won't on 3.3.8 and 3.3.9 ( which is latest at the moment ).

### Logged Error Messages

```text
Traceback (most recent call last):
  File "/app/utils.py", line 296, in get_user_agent
    driver = get_webdriver()
             ^^^^^^^^^^^^^^^
  File "/app/utils.py", line 177, in get_webdriver
    driver = uc.Chrome(options=options, browser_executable_path=browser_executable_path,
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/undetected_chromedriver/__init__.py", line 252, in __init__
    self.patcher = Patcher(
                   ^^^^^^^^
  File "/app/undetected_chromedriver/patcher.py", line 67, in __init__
    version_main_int = int(version_main)
                       ^^^^^^^^^^^^^^^^^
TypeError: int() argument must be a string, a bytes-like object or a real number, not 'NoneType'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/app/flaresolverr.py", line 105, in <module>
    flaresolverr_service.test_browser_installation()
  File "/app/flaresolverr_service.py", line 72, in test_browser_installation
    user_agent = utils.get_user_agent()
                 ^^^^^^^^^^^^^^^^^^^^^^
  File "/app/utils.py", line 302, in get_user_agent
    raise Exception("Error getting browser User-Agent. " + str(e))
Exception: Error getting browser User-Agent. int() argument must be a string, a bytes-like object or a real number, not 'NoneType'
Traceback (most recent call last):
  File "/app/utils.py", line 296, in get_user_agent
    driver = get_webdriver()
             ^^^^^^^^^^^^^^^
  File "/app/utils.py", line 177, in get_webdriver
    driver = uc.Chrome(options=options, browser_executable_path=browser_executable_path,
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/undetected_chromedriver/__init__.py", line 252, in __init__
    self.patcher = Patcher(
                   ^^^^^^^^
  File "/app/undetected_chromedriver/patcher.py", line 67, in __init__
    version_main_int = int(version_main)
                       ^^^^^^^^^^^^^^^^^
TypeError: int() argument must be a string, a bytes-like object or a real number, not 'NoneType'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/app/flaresolverr.py", line 105, in <module>
    flaresolverr_service.test_browser_installation()
  File "/app/flaresolverr_service.py", line 72, in test_browser_installation
    user_agent = utils.get_user_agent()
                 ^^^^^^^^^^^^^^^^^^^^^^
  File "/app/utils.py", line 302, in get_user_agent
    raise Exception("Error getting browser User-Agent. " + str(e))
Exception: Error getting browser User-Agent. int() argument must be a string, a bytes-like object or a real number, not 'NoneType'
```

**Comment by ilike2burnthing** [comment]
That's the error from v3.3.8, which was resolved in v3.3.9 - https://github.com/FlareSolverr/FlareSolverr/issues/962.

Restart the container and provide the start of the log.

**Comment by MervinJMinky** [comment]
You are right, my bad.
I do not know why it didn't pull 3.3.9 with the latest tag when i tried it earlier.
Problem solved !
